README.md

```diff
@@ -150,6 +151,37 @@ get its `Comment`s as well.
 The `url` declaration describes which named routes to use while generating URLs
 for your JSON. Not every adapter will require URLs.
 
+## Caching
+
+To cache a serializer, call ```cache``` and pass its options.
+The options are the same options of ```ActiveSupport::Cache::Store```, plus
+a ```key``` option that will be the prefix of the object cache
+on a pattern ```"#{key}-#{object.id}"```.
+
+**[NOTE] Every object is individually cached.**
+**[NOTE] The cache isn't automatically expired after update an object.**
+
+```ruby
+cache(options = nil) # options: ```{key, expires_in, compress, force, race_condition_ttl}```
+```
+
+Take the example bellow:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  cache key: 'post', expires_in: 3.hours
```

Comment:
@joaomdmoura could we make `key` optional? Then we could use `object.cache_key` by default. Makes sense?